dependencies.properties

```diff
@@ -8,7 +8,7 @@
 
 # Versions only, for dependencies which actual artifacts differ between Bazel and Gradle
 version.com_google_protobuf=3.9.1
-version.google_java_format=1.8
+version.google_java_format=1.7
```

Comment:
Maybe I missed some context. Why we are dropping down from 1.8 to 1.7?

Reply:
+1.

Reply:
as said in the description, version of google-java-format is downgraded to 1.7 because 1.8 supports java 11 minimum, while our JRE is java 8.